src/app/frontend/common/components/labels/labels_controller.js

```diff
@@ -19,6 +19,12 @@
 const alwaysVisibleLabelsNumber = 5;
 
 /**
+ * Regular expression matching URLs.
+ * @type {RegExp}
+ */
+const urlRegexp = new RegExp('(https?|ftp)://(-\.)?([^\s/?\.#-]+\.?)+(/[^\s]*)?', 'i');
```

Comment:
should we anchor the regexp? since we don't try to extract the actual url, and just make a href of the whole annotation